### ISSUE TITLE
Weird sounds when app is on background (iOS)

### ISSUE THREAD

**Issue by miguelpruivo** [luanpotter/audioplayers#564]
Hi, no matter how many sounds you have in your app, it looks like when it enters in background (just by going to the home screen on iOS) and it will start messing with your MacOS sound (if you are playing a music for example) and playing ticking sounds until you either relaunch the app or kill it.

Looks like a memory leak to me and happens _every_ time.

Thanks!

**Comment by muarachmann** [comment]
I think this happens on the emulator only... I have had this issue too, but on physical device it works just well.

**Comment by miguelpruivo** [comment]
Yes. It’s only on the emulator, however I wonder why it does happen.
